libnavigation-base/src/main/java/com/mapbox/navigation/base/options/OnboardRouterEndpointOptions.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapbox.navigation.base.options
+
+import android.content.Context
+import java.io.File
+import java.net.URI
+
+/**
+ * Defines endpoint's properties to retrieve routing tiles.
+ *
+ * @param tilesUri tiles endpoint
+ * @param version version of tiles
+ * @param userAgent HttpClient UserAgent
+ * @param builder used for updating options
+ */
+data class OnboardRouterEndpointOptions(
+    val tilesUri: URI,
+    val version: String,
+    val userAgent: String,
```

Comment:
Not sure if this should be exposed or handled internally (even if we have a default) 🤔

Reply:
Yeah is this a valid option? Happy to turn it into a constant

Reply:
I believe this could be an option but definitely something that we should control internally. Maybe for now we can keep it constant as NN is handling the requests. At the moment in which the requests are handled in the platform side we might need to pass in something different. Does that make sense?

Reply:
> Does that make sense?

Maybe, this change is only moving it. It had a default value before, and it has a default value now - not changing it here. But the default is in the builder https://github.com/mapbox/mapbox-navigation-android/pull/3020/files#diff-1ad3aac9729b5bcc432e2a46fc0f0e57R40

Question is if we should remove it as an option externally. Which would be moving it out of the constructor

Reply:
also could move it to the one place it's used, `MapboxOnboardRouter`